Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -724,6 +724,28 @@ ErrorOr<void> MainWidget::initialize_menubar(GUI::Window& window)
             }
         })));
 
+    TRY(m_layer_menu->try_add_action(GUI::Action::create(
+        "Duplicate Layer", { Mod_Ctrl | Mod_Shift, Key_D }, g_icon_bag.new_layer, [&](auto&) {
+            auto* editor = current_image_editor();
+            VERIFY(editor);
+            auto* active_layer = editor->active_layer();
+            if (!active_layer)
+                return;
```

Comment:
not sure how it is done with other layer commands but it feels you should have this action disabled in ui if thereʼs no active layer at the moment (on the other hand, if it canʼt happen, then this check seems superfluous).

Reply:
This currently follows the same pattern as other layer actions. I agree it would be nice to disable the action if we know it won't do anything, but I think it would be better to make that change to all layer based actions at the same time. That way the behavior of all menu items stays consistent.

Reply:
sure thing; anything that makes that consistent (thought it could be an accidental omission!). 👌